lua/apisix/plugins/zipkin.lua

```diff
@@ -131,6 +144,9 @@ function _M.access(conf, ctx)
     opentracing.proxy_span = opentracing.request_span:start_child_span(
             "apisix.proxy", ctx.ACCESS_END_TIME)
 
+    opentracing.proxy_span:set_tag("peer.service", conf.service_name)
```

Comment:
do we need a  test case to confirm this?

Reply:
sure I am adding more test cases